src/main/java/org/terasology/metalrenegades/world/dynamic/discoverables/ChestFillingSystem.java

```diff
@@ -112,15 +109,22 @@ private boolean slotContainsItem() {
      * @return The chest slot item.
      */
     private EntityRef getSlotItem() {
-        EntityRef item;
-        if (random.nextFloat() < BULK_SINGLE_RATIO) {
-            item = entityManager.create(bulkItems[random.nextInt(bulkItems.length)]);
-            ItemComponent itemComponent = item.getComponent(ItemComponent.class);
-            itemComponent.stackCount = 5;
-            item.saveComponent(itemComponent);
-        } else {
-            item = entityManager.create(singleItems[random.nextInt(singleItems.length)]);
+        if (hiddenItemsRegistry.isEmpty()) {
+            initialise();
```

Comment:
`initialise` is guaranteed to run before this happens. Print out the error without attempting again?